jdi-bdd-tests/src/test/java/cucumberTests/RunTest.java

```diff
@@ -0,0 +1,22 @@
+package cucumberTests;
+
+import cucumber.api.CucumberOptions;
+import cucumber.api.junit.Cucumber;
+import org.junit.AfterClass;
+import org.junit.runner.RunWith;
+
+import static com.epam.jdi.light.driver.WebDriverUtils.killAllSeleniumDrivers;
+
+@RunWith(Cucumber.class)
+@CucumberOptions(
+        features = "classpath:features"
+        , glue = {"com.epam.jdi.bdd", "cucumberTests"}
+        , plugin = {"pretty"}
+//        , tags = {"@textarea"}
+)
+public class RunTest {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'RunTest' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5001515)